arkouda/groupbyclass.py

```diff
@@ -263,9 +263,12 @@ def __init__(
                 self.keys, return_groups=True, assume_sorted=self.assume_sorted
             )
 
-        self.size = self.permutation.size
+        self.length = self.permutation.size
         self.ngroups = self.segments.size
 
+    def size(self):
+        return self.count()
+
     def count(self) -> Tuple[groupable, pdarray]:
```

Comment:
Missing the `@typechecked`. Since we are in the file and modifying this functionality, we should add it here.

Reply:
Done